### PR TITLE
Tutorial 2019 04

### PR DESCRIPTION
**Pull request by d-tomerini**
Updated the bands notebook to the latest commits of aiida_core and qe plugin. 
Thanks to @borellim for his help!
There is only the structure as requirement for the band (provided that the v1.1 of the SSSP upf 
precision are installed on the machine).

As in the commit, the band calculation of GaAs requires a change to finish correctly (either SCF option for time increased from 1800s, switch to 'fast' option on the protocol, or modification of the minimum time in the QE plugin)

### CODE REVIEW

**Comment by borellim** [comment]
Nice! Thanks @d-tomerini !

File: tutorial/pw/bandstructure/bandstructure.ipynb [hunk 1]
```diff
@@ -54,16 +55,13 @@
...
+   "metadata": {},
    "outputs": [],
    "source": [
     "# Loading the COD importer so we can directly import structure from COD id's\n",
-    "CodImporter = DbImporterFactory('cod')\n",
-    "importer = CodImporter()\n",
+    "importer = CodDbImporter()\n",
     "# Make sure here to define the correct codename that corresponds to the pw.x code installed on your machine of choice\n",
-    "codename = 'qe-pw-6.2.1@localhost'\n",
+    "codename = 'qe-6.3-pw@localhost'\n",
```
Comment (borellim) [inline tutorial/pw/bandstructure/bandstructure.ipynb]: @ltalirz is this the version of QE that they will have, and the name with which the pw code will be installed?

File: tutorial/pw/bandstructure/bandstructure.ipynb [hunk 2]
```diff
@@ -77,18 +75,18 @@
...
+    "\n",
     "structure_Al.get_formula()\n",
+    "\n",
     "# The following structure can be used instead of Al, but will take much longer on the AWS machine.\n",
-    "# GaAs COD ID='9008845'\n",
-    "# CaF2 COD ID='1000043'\n",
-    "# h-BN COD ID='9008997'"
+    "# CaF2 COD ID='1000043' -- approximately 1/2 hour to run\n",
+    "# h-BN COD ID='9008997' -- approximately 45 mins to run\n",
+    "# GaAs COD ID='9008845' -- approximately 2 hours to run "
```
Comment (borellim) [inline tutorial/pw/bandstructure/bandstructure.ipynb]: @d-tomerini are these times tested on the AWS VM or on your machine?

Comment (d-tomerini) [reply tutorial/pw/bandstructure/bandstructure.ipynb]: @borellim I put slightly longer times than those I got on my machine (that I assume it is faster than what the students will have access to). I used 1 processor in my tests.

File: tutorial/pw/bandstructure/bandstructure.ipynb (hunk 1 above)
Comment (ltalirz) [reply tutorial/pw/bandstructure/bandstructure.ipynb]: yes.

**Comment by ltalirz** [comment]
Thanks! 
Instead of updating the notebook here, could you please add it [here](https://github.com/aiidateam/aiida-tutorials/tree/tutorial_2019_05/docs/pages/2019_MARVEL_Psik_MaX/notebooks) ?

We will remove the tutorial-related notebooks from the demos repo.

**Comment by d-tomerini** [comment]
Sorry, I missed there was an organization change. 

Would you like it as a directory pw/bandstructure/ , or directly in notebooks/ as bandstructure.ipynb?

**Pull request merged**